src/main/resources/org/jpeek/metrics/LCC.xsl

```diff
@@ -60,9 +60,9 @@ SOFTWARE.
     </xsl:variable>
     <!--
     @todo #9: `indirectly-related-pairs` are pairs of methods, which are through other directly connected methods.
```

Comment:
@skapral also add the estimation: ``@todo #9:30min [...]``. For the next puzzle as well.

Reply:
@amihaiemil fixed for LCC and TCC descriptors.